src/MultiFormGoals/resources/js/blocks/multi-form-goal/index.js

```diff
@@ -23,12 +23,7 @@ const blockTemplate = [
 	[ 'core/media-text', {
 		imageFill: true,
 	}, [
-		[ 'core/heading', {
-			placeholder: __( 'Heading', 'give' ),
-		} ],
-		[ 'core/paragraph', {
-			placeholder: __( 'Summary', 'give' ),
-		} ],
+		[ 'give/multi-form-goal-content', {} ],
```

Comment:
@henryholtgeerts should the new "content" block wrap the `core/media-text` block as well? Or are we being intentionally opinionated with having an image?